scripts/r/07_team-sds.Rmd

```diff
@@ -794,7 +796,6 @@ final_teams %>% filter(animal == "petauroides_fistulator") %>% pull(outcome) ==
 #  end = 
 #  )
 
-# Super long because somebody refuses to use `here` and I don't want to modify `refit_sc.Rmd`
```

Comment:
salty 🧂

Reply:
I had to copy and paste because the knit hook from your document doesn't help when source code from outside... `here` would have solved this... just sayin

Reply:
If you'd use the knitr old way, it would work. The problem is that you are using in this Rmd file an approach that works well with quarto but not with Rmd... but it seems that I have failed to make this clear because we keep hitting this wall. :D

Reply:
::insert donald trump gif saying "wrong"::